packages/block-library/src/navigation-link/block.json

```diff
@@ -1,9 +1,6 @@
 {
 	"name": "core/navigation-link",
 	"category": "layout",
-	"parent": [
-		"core/navigation"
-	],
```

Comment:
Does this mean we'll be able to add navigation-link blocks to other parents?

Reply:
Yes, I could not find a way around this, see the third dot point under **Outstanding questions**.